acme.go

```diff
@@ -238,6 +245,30 @@ func acceptChallenge(client *http.Client, config *Config, challenge Challenge) (
 	return chalResp, nil
 }
 
+// pollAuthorization checks the current status of an authorization request.
+//
+// If client argument is nil, DefaultClient will be used.
+func pollAuthorization(client *http.Client, config *Config, url string) (Authorization, error) {
```

Comment:
Why do you need `config` here?

Also, I'd call it `pollAuthz` - shorter and still consistent with their naming as they use `authz` in other places.